### PR TITLE
Create Payouts page according issue 1070

### PR DESCRIPTION
**Pull request by Bangalisch**
Created an initial page for the Payouts.
Might need some more visuals, didn't really know what to put as its a pretty "dull" tab.

### CODE REVIEW

File: docs/Payouts.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Payouts
+
+Tied into the [Pull Payments](./PullPayments.md) we have the **payouts** tab. 
+In this tab, you will be able to process the Pull payment requests and think about Refunds, salary payouts, or withdrawals.
```
Comment (pavlenex) [inline docs/Payouts.md:4]:
```suggestion
This feature allows you to process pull payment (refunds, salary payouts, or withdrawals). 
```

File: docs/Payouts.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# Payouts
+
+Tied into the [Pull Payments](./PullPayments.md) we have the **payouts** tab. 
+In this tab, you will be able to process the Pull payment requests and think about Refunds, salary payouts, or withdrawals. 
+
+## How does it work?
```
Comment (pavlenex) [inline docs/Payouts.md:6]: Doesn't explain how payouts work cc @Kukks can you help us with defining this?

File: docs/Payouts.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+# Payouts
+
+Tied into the [Pull Payments](./PullPayments.md) we have the **payouts** tab. 
+In this tab, you will be able to process the Pull payment requests and think about Refunds, salary payouts, or withdrawals. 
+
+## How does it work? 
+
+We will go through two examples, one will be a Refund, and the other will be a salary payout. 
+As of writing this guide (v1.4.7), there is no automation for the last example.
```
Comment (pavlenex) [inline docs/Payouts.md:9]: Not relevant yet.
```suggestion
```

File: docs/Payouts.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+Tied into the [Pull Payments](./PullPayments.md) we have the **payouts** tab. 
+In this tab, you will be able to process the Pull payment requests and think about Refunds, salary payouts, or withdrawals. 
+
+## How does it work? 
+
+We will go through two examples, one will be a Refund, and the other will be a salary payout. 
+As of writing this guide (v1.4.7), there is no automation for the last example. 
+
+### Example
+Let's start with the Refund example.
```
Comment (pavlenex) [inline docs/Payouts.md:12]:
```suggestion
Let's start with the refund example. 
```

File: docs/Payouts.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+In this tab, you will be able to process the Pull payment requests and think about Refunds, salary payouts, or withdrawals. 
+
+## How does it work? 
+
+We will go through two examples, one will be a Refund, and the other will be a salary payout. 
+As of writing this guide (v1.4.7), there is no automation for the last example. 
+
+### Example
+Let's start with the Refund example. 
+The Customer has bought an item in your store but sadly has to return the item. He wants a refund.
```
Comment (pavlenex) [inline docs/Payouts.md:13]:
```suggestion
The customer has bought an item in your store but sadly has to return the item. They want a refund. 
```

File: docs/Payouts.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+## How does it work? 
+
+We will go through two examples, one will be a Refund, and the other will be a salary payout. 
+As of writing this guide (v1.4.7), there is no automation for the last example. 
+
+### Example
+Let's start with the Refund example. 
+The Customer has bought an item in your store but sadly has to return the item. He wants a refund. 
+Within BTCPay, the merchant creates a [Refund](./Refund.md) and provides the Customer with the appropriate link to claim his funds.
```
Comment (pavlenex) [inline docs/Payouts.md:14]:
```suggestion
Within BTCPay, you can create a [Refund](./Refund.md) and provide the customer with the link to claim their funds. 
```

File: docs/Payouts.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+## How does it work? 
+
+We will go through two examples, one will be a Refund, and the other will be a salary payout. 
+As of writing this guide (v1.4.7), there is no automation for the last example. 
+
+### Example
+Let's start with the Refund example. 
+The Customer has bought an item in your store but sadly has to return the item. He wants a refund. 
+Within BTCPay, the merchant creates a [Refund](./Refund.md) and provides the Customer with the appropriate link to claim his funds. 
+Whenever the Customer has given his address and claimed the funds, the merchant now has an entry in Payouts.
```
Comment (pavlenex) [inline docs/Payouts.md:15]:
```suggestion
Whenever the customer has given their address and claimed the funds, it will be shown in the `Payouts`. 
```

**Review by pavlenex — COMMENTED** [review]
I started reviewing the doc, but it has lots of fluff, too many text that explain concepts that I don't think we need to go in-depth into. We need to review this PR on a doc call to see how we may want to go about it.

File: docs/.vuepress/config.js [hunk 8]
```diff
@@ -75,6 +75,7 @@ const sidebarUserGuide = [
           ['/Refund', 'Refunds']
         ]
       },
+      ['/Payouts', 'Payouts'],
```
[the remark this answers is not in the document]
Comment (Bangalisch) [reply docs/.vuepress/config.js:78]: i think i would like to restructure fully in another PR. 
There's more places that need re-arrangement and some shuffling around. 
Will take this one on right away :)

**Comment by Bangalisch** [comment]
> I started reviewing the doc, but it has lots of fluff, too many text that explain concepts that I don't think we need to go in-depth into. We need to review this PR on a doc call to see how we may want to go about it.

There might be some fluff, tho there's no place so far that explains it. 
Hence why I maybe over did it a little, we'll put it in the Agenda for next call. 

I also think that as I wrote on the comment on ordering, we should create a task for it. 
There's a little more places that could use some ordering, did very little this time, to represent current GUI a little bit better.

**Review by Mjoly1024 — APPROVED** [review]
The refund was correct and approved

**Pull request merged**